use-cases/client-server-msgs-perf/svmsg_file_server.c

```diff
@@ -107,7 +108,13 @@ Client_info ActiveClients[MAX_CLIENTS];
  * Configuration for each server thread.
  */
 typedef struct {
-    int     server_id;  // Server's msgq-ID.
+    int         server_id;  // Server's msgq-ID.
+
+    // On Mac/OSX, pthread_self() returns a struct, not an ID. So, to get
+    // code compiling, print the thread's index in the threads[] array.
+    int         thread_idx;
+    uint64_t    num_ops;    // Processed by this thread
+
```

Comment:
Thread config struct changed so we can compile code on Mac.
Also on L 116, report back the # of messages processed by each thread.

In summary report, this field is reported as avg. # of messages / thread -- which shows us quickly whether messages are being distributed reasonably uniformly across server threads.